mystery_word.py

```diff
@@ -0,0 +1,201 @@
+import random
+
+
+guesses = 8
+
+
+def mystery_game_process(first_time=True):
+    """This function triggers the mystery_game. A user is presented
+    with a random word. The user is then shown the word with blank spaces
+    and told how many chars there are. Then the user must begin guessing."""
+
+    if first_time:
+        help_docs()
+
+    word_answer = get_random_word(user_difficulty_choice())
+
+    partial_answer_string = fancy_view_word(word_answer)
+    print("  {}      {} guesses left.".format(partial_answer_string,
+                                              guesses))
+    user_guess_process(word_answer)
+
+    return True
+
+
+def user_difficulty_choice():
+    """This function asks the user what difficulty they'd like."""
+
+    difficulty = 0
+
+    try:
+        difficulty = int(input("""
+        What difficulty would you like to play at:
+        Type '1' for easy
+        Type '2' for medium
+        Type '3' for hard
+        Type difficulty level here -->> """))
+    except:
+        print("Please only enter 1, 2, or 3 for difficulty.")
+        return user_difficulty_choice()
+    return difficulty
+
+
+def get_random_word(difficulty):
```

Comment:
This function does too much. It should be split into two functions -- one to get all the words and one to filter them.